src/components/Register.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { useState } from "react";
+
+const apiUrl = "http://localhost:4000";
+
+export default function Register() {
+  const [user, setUser] = useState({
+    username: "",
+    password: "",
+  });
+  const [registerResponse, setRegisterResponse] = useState("");
+
+  function handleChange(event) {
+    const { name, value } = event.target;
+    setUser((prevUser) => {
+      return {
+        ...prevUser,
+        [name]: value,
+      };
+    });
```

Comment:
Hmm interesting I haven't seen something like this until now, is it not possible to remove the return keyword and have the function call and object in a single line?